default.nix

```diff
@@ -62,6 +66,8 @@ let
 
       matchDot = mkMatcher ''([.])([${notInSymbol}]|$).*'' 2;
 
+      matchFunction = throw "matchFunction: Not implemented";
```

Comment:
Note that `matchFunction` is already not implemented, it was just hard to notice because of `with; ...`.